oak-authorization-principalbased/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/principalbased/impl/FilterProviderImpl.java

```diff
@@ -98,19 +107,23 @@ public Filter getFilter(@NotNull SecurityProvider securityProvider, @NotNull Roo
 
     //----------------------------------------------------< SCR Integration >---
 
+    public FilterProviderImpl() {
+        // constructor to use from SCR (not yet possible to use constructor injection, see https://issues.apache.org/jira/browse/OAK-9837)
+    }
+
     @Activate
-    protected void activate(Configuration configuration, Map<String, Object> properties) {
-        setPath(configuration);
+    protected void activate(Configuration configuration) {
```

Comment:
removing the unused property map is an unrelated change, right? i would not do it as part of this PR but create a separate task for that.

Reply:
Sorry, I will revert if you insist but will not do the additional overhead of creating another PR. For me things like these should be fixed along with the issues. In this case I fail to see risk of regression.